sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/sweep.py

```diff
@@ -149,6 +150,30 @@ def trial(self):
         """Id or instance of the command component/job to be run for the step."""
         return self._component
 
+    @property
+    def search_space(self):
+        return self._search_space
+
+    @search_space.setter
+    def search_space(self, values: Dict[str, Dict[str, Union[str, int, float, dict]]]):
+        """Support using dict to initialize search_space attribute value."""
```

Comment:
No need to have docstring for setter, please do it for getter (which will be generated in reference doc site).

Reply:
deleted